mkosi

```diff
@@ -605,7 +605,7 @@ def mount_cache(args, workspace):
 
 def umount(where):
     # Ignore failures and error messages
-    subprocess.run(["umount", "-n", where], stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL)
+    subprocess.run(["umount", "-n", where], stdout=subprocess.DEVNULL)
```

Comment:
we ignore errors on this one for a reason too: we call it for stuff that might very well not be a mount point, and we let umount figure that out, but are not interested in its error message...